libtenzir/builtins/connectors/email.cpp

```diff
@@ -68,19 +69,30 @@ class saver final : public plugin_saver {
     if (auto err = to_error(tx.handle().set(CURLOPT_UPLOAD, 1))) {
       return err;
     }
-    auto code = tx.handle().set(CURLOPT_URL, args_.endpoint);
-    if (code != curl::easy::code::ok) {
-      auto err = to_error(code);
-      diagnostic::error("failed to set SMTP server request")
-        .note("server: {}", args_.endpoint)
+    if (auto err = tx.prepare(std::move(args_.endpoint))) {
+      diagnostic::error("failed to prepare SMTP server request")
+        .note("{}", err)
+        .emit(ctrl.diagnostics());
+      return err;
+    }
```

Comment:
The `prepare` call must come before any other option setting calls to `tx`.
Move it above the `CURLOPT_UPLOAD` setting to restore compatibility with servers that don't implement VRFY.

Reply:
Moved up.